packages/app/src/components/uiBlocks/TagEntry.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="flex flex-col flex-shrink-0">
+    <div class="relative pb-1/1">
+      <img class="absolute w-full h-full rounded-xl" :src="image" />
+    </div>
+    <div id="TagName" class="h-2/6 text-white font-semibold" :class="[`text-${textSize}`]">
+      {{ name }}
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  props: {
+    img: {
+      type: String,
+      required: true,
+      default: '',
+    },
+    name: {
+      type: String,
+      required: true,
+    },
+    textSize: {
+      type: String,
+      required: false,
+      default: 'base',
+    },
+  },
+  setup(props) {
+    const image = `./${props.img}`;
```

Comment:
```suggestion
    const image = ref<string>(`./${props.img}`);  
```


shouldn't that be a ref?

Reply:
I think it's not important because, the image can not really change in realtime, but it won't hurt to give it a ref

Reply:
fixed in 3be7fb18d799aa08914764c9f315d65ff5a59db5